daphne/src/aborts.rs

```diff
@@ -252,8 +254,8 @@ impl DapAbort {
             Self::UnrecognizedAggregationJob { .. } => {
                 ("Unrecognized aggregation job", Some(self.to_string()))
             }
-            Self::UnrecognizedMessage => {
-                ("Failed to parse the request body", Some(self.to_string()))
+            Self::UnrecognizedMessage { .. } => {
+                ("Malformed or unexpected message", Some(self.to_string()))
```

Comment:
This language better matches what we'll see in the next version of the spec: https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/blob/main/draft-ietf-ppm-dap.md?plain=1#L502
```suggestion
                ("Malformed or invalid message", Some(self.to_string()))
```